docs/site/output.md

```diff
@@ -11,11 +11,12 @@ Calva categorizes output into three types:
 * **evaluation output**: stdout/stderr from an evaluation
 * **other output**: Other messages, logs, etc
 
-With the setting `calva.outputDestinations`, you can configure where each category of output should go to:
+With the setting `calva.outputDestinations`, you can configure where each category of output should go to. These are the allowed values and their descriptions:
 
-* the [REPL Window](repl-window.md)
-* the _Calva Says_ Output Channel
-* the _Calva Output_ (pseudo) Terminal
+* `"repl-window"` - The [REPL Window](repl-window.md) (an editor-based read/write output view). This is the default value.
+* `"webview"` - The [output webview](output-webview.md) (a read-only webview that is much more performant than the REPL Window)
```

Comment:
I wonder... `webview` is an implementation detail. May lead some people to think that it is external to VS Code. How about we just call it `output-window`? It will create some confusion in the code, but we can clean that up in a later commit.

Reply:
I like this idea. I'll do that.